README.md

```diff
@@ -36,6 +36,17 @@ Make sure this repository is in your `load-path`, and then:
 
 ## Configuration
 
+### Automatically turn on completions
+
+Completions will not be loaded automatically in `Swift` mode. To enable them, add the following to your emacs init file *(use `SPC-f-e-i` to open it in spacemacs)*:
+```elisp
+  (add-hook 'swift-mode-hook (lambda ()
```

Comment:
Hmm could you remove the preceding whitespace, also personally I use:

```elisp
(add-to-list 'company-backends 'company-sourcekit)
```

However that depends on your list of backends all checking the mode and passing through correctly (vs. short-circuiting)